datasets/br/suspended_companies/crawler.py

```diff
@@ -0,0 +1,111 @@
+    The CSV URL is dynamically generated and changes every day
+    and is created by concatenating the SITE_URL with the date in the format YYYYMMDD.
+
+    The date is in a script tag in the main page.
+
+    :param context: The context object.
+
+    :return: The URL of the CSV file.
+    """
+
+    # The header is necessary because the website blocks requests without a user agent.
+    doc = context.fetch_html(SITE_URL, headers=HEADERS)
+    path = '//script'
+    date_pattern = re.compile(r'"ano"\s*:\s*"(\d+)",\s*"mes"\s*:\s*"(\d+)",\s*"dia"\s*:\s*"(\d+)"')
+    for script in doc.xpath(path):
+        if script.text:
+            match = date_pattern.search(script.text)
+            if match:
+                year, month, day = match.groups()
+                return SITE_URL + f"/{year}{month}{day}"
+
+    raise ValueError("Data URL not found")
+
+
+def get_data(csv_url: str, context: Context) -> List[dict]:
+    """
+    Fetches the CSV file as a zip from the website decompresses it and parses it using the csv library
+    and returns the data as a list of dicts.
+
+    :param csv_url: The URL of the CSV file.
+    :param context: The context object.
+
+    :return: The data fetched from the website as a list of dicts.
+    """
+    # The header is necessary because the website blocks requests without a user agent.
+    response = context.fetch_response(csv_url, headers=HEADERS)
+    zip_file = ZipFile(io.BytesIO(response.content))
+    file_name = zip_file.namelist()[0]
+
+    csv_str = zip_file.read(file_name).decode('iso-8859-1')
+    lines = csv_str.splitlines()
+
+    reader = csv.DictReader(lines, delimiter=';')
+
+    return [row for row in reader]
+
+
+def create_entities(data: List[dict], context: Context) -> None:
+    """
+    Creates entities from the data fetched from the website.
+
+    :param data: The data fetched from the website as a list of dicts.
+    :param context: The context object.
+    """
+
+    for raw_entity in data:
+        if raw_entity['TIPO DE PESSOA'] == 'F':
+            entity = context.make('Person')
+            entity.id = context.make_id(raw_entity['CPF OU CNPJ DO SANCIONADO'])
+            entity.add('name', raw_entity['NOME DO SANCIONADO'])
+            entity.add('country', 'br')
+            entity.add('taxNumber', raw_entity['CPF OU CNPJ DO SANCIONADO'])
+
+            context.emit(entity, target=True)
+
+        if raw_entity['TIPO DE PESSOA'] == 'J':
+            entity = context.make('Company')
+            entity.id = context.make_id(raw_entity['CPF OU CNPJ DO SANCIONADO'])
+            entity.add('name', raw_entity['NOME DO SANCIONADO'])
+            entity.add('country', 'br')
+            entity.add('taxNumber', raw_entity['CPF OU CNPJ DO SANCIONADO'])
+
+            context.emit(entity, target=True)
+
+        sanction = h.make_sanction(context, entity)
+        sanction.add("program", "Brazil disreputed and sanctioned companies")
+        sanction.add("country", "br")
```

Comment:
You can also set `country` here by adding it to the `publisher` section in the metadata.